business-central-tests/business-central-tests-rest/src/test/java/org/kie/wb/test/rest/functional/SpaceIntegrationTest.java

```diff
@@ -75,6 +76,7 @@ public void testCreateMinimal() {
         testCreate(space);
     }
 
+    @Ignore
```

Comment:
please add here a reason and jira URL you created for fixing the tests in future